articles/integrating-b2c-mpesa-api-using-node-js/index.md

```diff
@@ -0,0 +1,299 @@
+### Integrating B2C Mpesa API using Node.js
+
+Mpesa can be described as a phone-based mobile banking service. Since its inauguration in 2007 by Vodafone Group PLC and Safaricom in Kenya, it has expanded to eight more countries in Africa.
+
+An API (Application Programming Interface) is an intermediary between two software applications that enables them to communicate.
+
+B2C Mpesa API enables software developers to integrate the functionality of merchants sending payments to their customers in their applications.
```

Comment:
Could you tell the readers that B2C stands for `Business to Consumer`

Reply:
done.